doc/decisions/database_structure.md

```diff
@@ -0,0 +1,95 @@
+# Database Structure
+
+The main goal of this document is to describe the database structure/layout especially in terms of relations between data entities.
+
+## Problem
+
+Semantically the plant information can be described as a hierarchy of entities.
+
+A plant can be classified
+- on variety level, and/or
+- on species level.
+
+Furthermore:
+- a variety belongs to a specie
+- specie belongs to a genus
+- genus belongs to a subfamily or family
+- subfamily belongs to a family.
+
+Please take a look at the diagram below.
```

Comment:
Please directly add it here. (Forward references are bad style, especially if they are unnamed.)